plugins/aggregators/basicstats/README.md

```diff
@@ -1,8 +1,8 @@
 # BasicStats Aggregator Plugin
 
 The BasicStats aggregator plugin gives count, diff, max, min, mean,
-non_negative_diff, sum, s2(variance), stdev for a set of values, emitting the
-aggregate every `period` seconds.
+non_negative_diff, sum, s2(variance), stdev for a set of values, last,
+first emitting the aggregate every `period` seconds.
```

Comment:
This reads a little poorly to me, thoughts on this?
```suggestion
non_negative_diff, sum, s2(variance), stdev for a set of values, last and
first, emitting the aggregate every `period` seconds.
```

Reply:
no opinion. @srebhan ?

Reply:
As Dane is the native speaker and I'm not,  I'd say we go with his suggestion. :-)

Reply:
@tguenneguez any chance you change the wording or should I commit the suggestion?

Reply:
Sorry I didn't see your previous answer.
As I'm French I'm not in a good position to make linguistic corrections ;-)